README.md

```diff
@@ -6,3 +6,31 @@
 [00]: https://github.com/amontalenti/pydata-book-club/blob/master/docs/00_python_setup.md
 [01]: https://github.com/amontalenti/pydata-book-club/blob/master/docs/01_pydata_env_setup.md
 [02]: https://github.com/amontalenti/pydata-book-club/blob/master/docs/02_pd_mpl.md
+
+[Makefile](Makefile) for using [Jupyter Notebook Data Science Stack](https://hub.docker.com/r/jupyter/datascience-notebook/)
```

Comment:
Haven't seen this docker image before, looks interesting. You might have to give me some time to review and try it out before I accept this PR.

Reply:
No problem, take your time to review and test. 

All commands are in root Makefile. `notebooks/Makefile` needs to be run in a jupyter docker container (via root Makefile).